### DISCUSSION TITLE
How can I bypass the cloudflare captchas?

### DISCUSSION THREAD

**Discussion by bookmateo** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#356] [Q&A]
hello could someone help with these. I have tried run the scrap several times but all it does is, it opens the browser logs me in to my account and then the page for browser checking appears next. so how can i bypass that page for cloudflare captchas?

**Comment by cullzie** [comment]
Hey @bookmateo, Are you using chrome based browser and the latest code?
